exercises/kindergarten-garden/canonical-data.json

```diff
@@ -0,0 +1,127 @@
+{
+  "exercise": "kindergarten-garden",
+  "version": "1.0.0",
+  "comments": [
+    "Garden can be created with just a diagram or a diagram and students.",
+    "Students' gardens are assigned in alphabetical order.",
+    "The default students are:",
+    "  Alice, Bob, Charlie, David, Eve, Fred, Ginny, Harriet, Ileana, Joseph, Kincaid, and Larry."
+  ],
+  "cases": [
+    {
+      "description": "partial garden tests",
+      "cases": [
+        {
+          "description": "test garden with single student",
```

Comment:
I agree. The "test" prefix doesn't add anything as the description is only valid in the context of a test case :)